huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go

```diff
@@ -484,6 +484,46 @@ func ResourceFgsFunction() *schema.Resource {
 					},
 				),
 			},
+			"enable_dynamic_memory": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: `Whether the dynamic memory configuration is enabled.`,
+			},
+			"is_stateful_function": {
+				Type:        schema.TypeBool,
+				Optional:    true,
```

Comment:
It's best to comment on the reason why you don't add the Computed attribute.

Reply:
because it can be closed.

Reply:
done, the comments are supported.